print-ip6opts.c

```diff
@@ -196,6 +196,13 @@ ip6_opt_process(netdissect_options *ndo, const u_char *bp, int len,
 		ND_PRINT(")");
 	    }
 	    break;
+	case IP6OPT_TPF:
+	    if (GET_U_1(bp + 1) != 4) {
```

Comment:
You're checking the length of the first option, no matter where this option is in the list.  Other option printers use `bp + i + 1` as the location of the length field.